tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -169,10 +169,24 @@ When('the user renames file/folder {string} to {string} using the webUI', functi
   return client.page.FilesPageElement.filesList().renameFile(fromName, toName)
 })
 
+When('the user tries to rename folder {string} to {string} using the webUI', function (fromName, toName) {
```

Comment:
Why having an extra step and not using the one below

Reply:
:hocho: